src/components/logic/calculate.js

```diff
@@ -0,0 +1,3 @@
+/* function (item) {
+   return !!item.match(/[0-9]+/);
+}; */
```

Comment:
- [ ] Kindly remove the commented code from your project, you can leave comments about your code but not commented code, this will make your codebase look neat and readable 😉

Reply:
- [ ] Please kindly remove this commented code as pointed earlier, this will make your codebase clean. 👍